repo/packages/M/minio/6/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "packagingVersion": "3.0",
+  "name": "Minio",
+  "version": "0.0.7-RELEASE.2017-05-05T01-14-51Z",
+  "minDcosReleaseVersion": "1.8",
+  "scm": "https://github.com/minio/minio.git",
+  "maintainer": "dev@minio.io",
+  "website": "https://www.minio.io",
+  "framework": false,
+  "description": "Minio is an object storage server for cloud native applications. To deploy at scale, use DC/OS to orchestrate Minio containers one per tenant.",
+  "tags": [
+    "data",
+    "storage",
+    "filesystem"
+  ],
+  "preInstallNotes": "There is no limit to number of Minio services you can launch.",
```

Comment:
@krisis this message is for the integration and not for the base tech and is required for all packages that are not in beta or selected. We can discuss offline on how to make it beta ready. Please add a link to the example and I can merge both the PRs at the same time. 
Other changes look good. Will merge the PR when these comments are addressed.